core/system/src/mpu/vmpu.c

```diff
@@ -332,6 +332,7 @@ static void vmpu_load_boxes(void)
 
     /* load box 0 */
     vmpu_load_box(0);
+    *(__uvisor_config.uvisor_box_context) = (uint32_t *) g_context_current_states[0].context;
```

Comment:
cc @AlessandroA ?
This is called from `uvisor_init`, so it might be to early to use unpriv write?